HISTORY.md

```diff
@@ -2,6 +2,9 @@
 ## Unreleased
 ### Public API Change
 * Support dynamically change `stats_dump_period_sec` option via SetDBOptions().
+* DB::Get in place of std::string accepts PinnableSlice, which avoids the extra memcpy of value to std::string in most of cases.
+** PinnableSlice releases the pinned resources that contain the value when it is destructed or when ::Reset() is called on it.
```

Comment:
I think we should not use the subitems like using **
I think it won't be readable when we do the release https://github.com/maysamyabandeh/rocksdb/blob/66174bc720a0e213f74f399cebc56bc247856d0f/HISTORY.md